packages/ui/src/proposals/modals/AddNewProposal/components/SpecificParameters/FundingRequest.tsx

```diff
@@ -17,22 +55,93 @@ export const FundingRequest = () => {
         </RowGapBlock>
       </Row>
       <Row>
-        <RowGapBlock gap={20}>
-          <InputComponent
-            label="Amount"
-            tight
-            units={CurrencyName.integerValue}
-            required
-            message="Amount must be greater than zero"
-            name="fundingRequest.amount"
-          >
-            <TokenInput id="amount-input" placeholder="0" name="fundingRequest.amount" />
-          </InputComponent>
-          <InputComponent label="Recipient account" required inputSize="l">
-            <SelectAccount name="fundingRequest.account" />
-          </InputComponent>
+        <RowGapBlock gap={payMultiple ? 6 : 24}>
+          <Row>
+            <InlineToggleWrap>
+              <Label>Pay multiple</Label>
+              <Tooltip
+                tooltipTitle="Pay multiple"
+                tooltipText="For multiple accounts and amounts, follow this CSV pattern:<br/> 
+              account1, amount1<br/> 
+              account2, amount2<br/> 
+              ...<br/> 
+              account20, amount20"
+              >
+                <TooltipDefault />
+              </Tooltip>
+            </InlineToggleWrap>
+            <ToggleCheckbox falseLabel="No" trueLabel="Yes" name="fundingRequest.payMultiple" id="pay-multiple" />
+          </Row>
+          {payMultiple && (
+            <Row>
+              <Prompt>
+                <TextSmall>
+                  For <TextInlineSmall bold>multiple accounts and amounts</TextInlineSmall>, follow this CSV pattern:
+                  <br />
+                  account1, amount1
+                  <br />
+                  account2, amount2
+                  <br />
+                  ...
+                  <br />
+                  account20, amount20
+                </TextSmall>
+              </Prompt>
+            </Row>
+          )}
         </RowGapBlock>
       </Row>
+      <Row>
+        {payMultiple ? (
+          <RowGapBlock gap={12}>
+            <InputComponent
+              label="Destination accounts and transfer amounts"
+              required
+              message={'You can select up to 20 recipients'}
+              name="fundingRequest.csvInput"
+              id="accounts-amounts"
+              inputSize="xl"
+            >
+              <InputTextarea
+                id="accounts-amounts"
+                name="fundingRequest.csvInput"
+                placeholder="Destination account address and amount"
+              />
+            </InputComponent>
+            <HiddenCheckBox name="fundingRequest.hasPreviewedInput" checked={hasPreviewedInput} />
+            {shouldPreview() && <ErrorPrompt>Please preview and validate the inputs to proceed</ErrorPrompt>}
```

Comment:
And this:
```suggestion
            {canPreviewInput && !hasPreviewedInput && <ErrorPrompt>Please preview and validate the inputs to proceed</ErrorPrompt>}
```
I might have missed something but AFAICS this should be enough.